src/scss/partials/_header.scss

```diff
@@ -70,7 +70,10 @@ $header--transparent-opacity        : 0.7;
         border: none;
         position: absolute;
         width: 100%;
+    }
 
+    .c-header--gradient,
+    %c-header--gradient {
```

Comment:
What's the reason this needs to be defined as a placeholder as well as just a classname? (just wondering where it's being used altogether)

Reply:
I've added it as a helper. In future, I will toggle the gradient using a class in the body, so I plan to reuse the these styles in my stylesheet.

Reply:
It should do all of this work from within the header component – so if there's a class that can toggle this state off, it should be added to the header component styles somewhere.

So this would become:

```scss
.c-header--gradient {
   .is-someClassToTurnHeaderTransparencyoff & {
   }
}
```

Would be best to add this when you need it though.